File: app/src/main/java/com/filemanager/fileexplorer/myfile/cleanmaster/free/adapter/MyViewHolder.java
package com.filemanager.fileexplorer.myfile.cleanmaster.free.adapter;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

import com.demo.example.R;


public class MyViewHolder extends RecyclerView.ViewHolder {
    TextView date;
    ImageView imageView;
    ImageView img_check;
    TextView time;
    TextView title;


    public MyViewHolder(View view) {
        super(view);
        this.date = (TextView) view.findViewById(R.id.date);
        this.time = (TextView) view.findViewById(R.id.time);
        this.title = (TextView) view.findViewById(R.id.title);
        this.imageView = (ImageView) view.findViewById(R.id.icon_file);
        this.img_check = (ImageView) view.findViewById(R.id.img_check);
    }
}

File: app/src/main/java/com/demo/filemanager/adapter/MyViewHolder.kt
package com.demo.filemanager.adapter

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.demo.filemanager.R

class MyViewHolder(view: View) : RecyclerView.ViewHolder(view) {
    var date: TextView
    var imageView: ImageView
    var img_check: ImageView
    var time: TextView
    var title: TextView

    init {
        date = view.findViewById<View>(R.id.date) as TextView
        time = view.findViewById<View>(R.id.time) as TextView
        title = view.findViewById<View>(R.id.title) as TextView
        imageView = view.findViewById<View>(R.id.icon_file) as ImageView
        img_check = view.findViewById<View>(R.id.img_check) as ImageView
    }
}
